### PR TITLE
Draft version of how to pass a DS to Jariko.

### PR DESCRIPTION
**Pull request by lanarimarco**
## Description

Draft version of how to pass a DS to Jariko.

## Checklist:
- [] There are tests regarding this feature
- [X] The code follows the Kotlin conventions (run `./gradlew ktlintCheck`)
- [X] The code passes all tests (run `./gradlew check`)
- [] There is a specific documentation in the `docs` directory

### CODE REVIEW

File: examples/src/main/kotlin/com/samples/jariko/RpgSamples.kt [hunk 1]
```diff
@@ -68,4 +70,58 @@ fun execWithCallBack(programSource: String, programArgs: List<String>, jarikoCal
...
+    dataStructValue.set(
+        field = FieldDefinition(
+            name = "Surname",
+            type = StringType(5, false),
+            explicitStartOffset = 5,
+            explicitEndOffset = 0
+        ),
+        value = StringValue("Smith")
+    )
+    // note rpgType = RpgType.ZONED is always required regardless of number type definition in rpg program
```
Comment (benetti-smeup) [inline examples/src/main/kotlin/com/samples/jariko/RpgSamples.kt:117]: Why should I use RpgType.ZONED for all kind of numeric type?

Comment (lanarimarco) [reply examples/src/main/kotlin/com/samples/jariko/RpgSamples.kt:117]: This is a very long story. I will try to explain to you.
Jariko only accepts a list of strings as input parameters.
First of all I had to find a way to represent a DS as a string.
I found that DataStructValue (the DataStruct object model in Jariko) is able to provide a good string representation of itself, via the DataStructValue.value function, only for numeric fields defined as RpgType.ZONED.
However we are only talking about a draft version, and these problems will be addressed in a deliverable, in fact this is not the solution but a possible solution.

Comment (benetti-smeup) [reply examples/src/main/kotlin/com/samples/jariko/RpgSamples.kt:117]: ok, thanks

**Review by benetti-smeup — APPROVED** [review]
ok

**Pull request merged**